frontend/src/Notifications/Notifications.jsx

```diff
@@ -28,6 +28,15 @@ export default function Notifications() {
         fetchNotifications();
     }, []);
 
+    useEffect(() => {
+        if (notification) {
+            setNotifications((prevNotifications) => [notification, ...prevNotifications]);
+            const storedNotifications = JSON.parse(localStorage.getItem('notifications')) || [];
```

Comment:
If we are storing notification in state, why do we also have to store it in localStorage?

Reply:
State updates reflect immediate changes in the UI, while local storage ensures notifications are retained even if the page is reloaded or the user goes offline.

Reply:
I also implemented the suggested helper function here: https://github.com/Synchronize-X/synchronize/pull/12/commits/0154232181cd658219da3de050b742c0e17074ff#diff-7b96395dc87816cc8a33aa58505374206f106851061f91f925ad3a677f6ed2f2R31

Reply:
are notifications not stored in a database?

Reply:
Yes, ordinary notifications are (by put requests and called back) but live notifications (web socket notifications) are not.